client/dashboard/utils/gridicons.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+	audio,
+	backup,
+	blockTable,
+	caution,
+	check,
+	cloud,
+	cog,
+	comment,
+	commentAuthorAvatar,
+	connection,
+	customPostType,
+	error,
+	homeButton,
+	image,
+	layout,
+	lock,
+	menu,
+	pages,
+	people,
+	postContent,
+	plugins,
+	receipt,
+	swatch,
+	trash,
+	update,
+	video,
+	wordpress,
+} from '@wordpress/icons';
+
+const icons: Record< string, React.ReactElement > = {
+	audio,
+	checkmark: check,
+	cart: receipt,
+	cloud,
+	cog,
+	comment,
+	'custom-post-type': customPostType,
+	history: backup,
+	image,
+	layout,
+	lock,
+	menu,
+	'multiple-users': people,
+	'my-sites': wordpress,
+	notice: caution,
+	posts: postContent,
+	pages,
+	plans: connection,
+	plugins,
+	science: swatch,
+	spam: error,
+	status: homeButton,
+	sync: update,
+	themes: blockTable,
+	trash,
+	user: commentAuthorAvatar,
+	video,
+};
+
+export function gridiconToWordPressIcon( slug: string ): React.ReactElement {
+	return icons[ slug ] ?? homeButton;
```

Comment:
Would it be better to log the error if the mapping cannot be found?

Reply:
We need to set up Sentry, which is not in the new dashboard yet; I think it needs to be a separate effort 🤔 (also the current logic is tied to the Redux store)

Reply:
We will notice it if we see the fallback empty icon

<img width="369" height="60" alt="Image" src="https://github.com/user-attachments/assets/e57511a4-3305-4bf4-9cd3-e327a9a55ce4" />

Reply:
But it might be the `status` grid icon

Reply:
`status` is actually already the fallback icon in Jetpack's activity log logic 😄 but anyway I updated the fallback for missing mapping just to be really sure:

<img width="365" height="61" alt="image" src="https://github.com/user-attachments/assets/fde9291a-3aa8-4544-8cda-dc2557a08b1d" />